caf-audit/README.md

```diff
@@ -139,134 +139,19 @@ A generated client-side library should be referenced in the normal way in the ap
 	    <version>1.0.0-SNAPSHOT</version>
 	</dependency>
 
-### ConfigurationSource
+#### Direct to Elasticsearch Configuration  
+Configuration required to be supplied via environment variable:  
+- *CAF_ELASTIC_HOST_AND_PORT_VALUES*: A comma separated list of hostnames and ports to use when contacting elasticsearch. eg. localhost:9200,otherHost:9200  
```

Comment:
Use code format (i.e. backticks) rather than italics for the environment variables.  I wonder would they be better in a table?